docs/setup/config.md

```diff
@@ -83,7 +83,8 @@ The default fraction for managed memory can be adjusted using the `taskmanager.m
 
 - `taskmanager.memory.segment-size`: The size of memory buffers used by the memory manager and the network stack in bytes (DEFAULT: 32768 (= 32 KiBytes)).
 
-- `taskmanager.memory.preallocate`: Can be either of `true` or `false`. Specifies whether task managers should allocate all managed memory when starting up. (DEFAULT: false)
+- `taskmanager.memory.preallocate`: Can be either of `true` or `false`. Specifies whether task managers should allocate all managed memory when starting up. (DEFAULT: false). When `taskmanager.memory.off-heap` is set to `true`, then it is advised that this configuration is also set to `true`, because when set to
+`false`, cleaning up of the allocated offheap memory kicks up only when the configured JVM parameter MaxDirectMemorySize is reached by triggering a full GC.
```

Comment:
"kicks up" --> "happens".

The reading flow is easier if this is split into two sentences (I would split at ", because")